infra/core/host/staticwebapp.bicep

```diff
@@ -13,7 +14,7 @@ resource web 'Microsoft.Web/staticSites@2022-03-01' = {
   tags: tags
   sku: sku
   properties: {
-    provider: 'SwaCli'
+    provider: 'Custom'
```

Comment:
Is this a typo?

Reply:
No, this is what they use now in all AZD templates. I think the SWACLI provided is set by the CLI directly when deployed